tools/blocksconvert/scanner/scanner.go

```diff
@@ -134,6 +144,14 @@ func NewScanner(cfg Config, scfg blocksconvert.SharedConfig, l log.Logger, reg p
 			Name: "cortex_blocksconvert_bigtable_parsed_index_entries_total",
 			Help: "Number of parsed index entries",
 		}),
+		currentTableRanges: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_blocksconvert_scanner_bigtable_ranges_in_current_table",
+			Help: "Number of ranges to scan from current table.",
+		}),
+		currentTableScannedRanges: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_blocksconvert_scanner_bigtable_scanned_ranges_from_current_table",
+			Help: "Number of scanned ranges from current table. (Goes from 0 on each new table)",
```

Comment:
> (Goes from 0 on each new table)

I find this a bit unclear. Maybe we could just say "Resets to 0 every time a table is getting scanned or its scan has completed."